soh/soh/OTRGlobals.cpp

```diff
@@ -1648,7 +1648,7 @@ extern "C" int CustomMessage_RetrieveIfExists(PlayState* play) {
             }
         } else if (Randomizer_GetSettingValue(RSK_DAMPES_DIARY_HINT) && textId == TEXT_DAMPES_DIARY) {
             messageEntry = CustomMessageManager::Instance->RetrieveMessage(Randomizer::randoMiscHintsTableID, TEXT_DAMPES_DIARY);
-        } else if (Randomizer_GetSettingValue(RSK_GREG_HINT) && (textId == 0x704C || textId == 0x6E || textId == 0x84)) {
+        } else if (Randomizer_GetSettingValue(RSK_GREG_HINT) && (textId == 0x704C || textId == 0x6E)) {
```

Comment:
I see there's 2 text ID's here, what's the difference between the 2?

Reply:
> I see there's 2 text ID's here, what's the difference between the 2?
They're both for the treasure minigame NPC
704C  is the "with that key you can xxxx"
6E is the you're a good gambler